src/components/tooltip/tooltip.spec.js

```diff
@@ -92,47 +120,86 @@ describe('MdTooltip Component', function() {
     );
 
     expect(element.attr('aria-label')).toBe('HELLO');
+    expect(element.attr('aria-labelledby')).toBeDefined();
   });
 
   it('should update the aria-label when the interpolated value changes',
       function() {
-        buildTooltip(
-          '<md-button>' +
-            '<md-tooltip>{{ testModel.ariaText }}</md-tooltip>' +
-          '</md-button>'
-        );
+    var ariaLabelledby;
+    var ariaLabelledby2;
 
-        $rootScope.$apply(function() {
-          $rootScope.testModel.ariaText = 'test 1';
-        });
+    buildTooltip(
+      '<md-button>' +
+        '<md-tooltip>{{ testModel.ariaText }}</md-tooltip>' +
+      '</md-button>'
+    );
 
-        expect(element.attr('aria-label')).toBe('test 1');
+    $rootScope.$apply(function() {
+      $rootScope.testModel.ariaText = 'test 1';
+    });
 
-        $rootScope.$apply(function() {
-          $rootScope.testModel.ariaText = 'test 2';
-        });
+    expect(element.attr('aria-label')).toBe('test 1');
+    ariaLabelledby = element.attr('aria-labelledby');
+    expect(ariaLabelledby).toBeDefined();
+
+    $rootScope.$apply(function() {
+      $rootScope.testModel.ariaText = 'test 2';
+    });
+
+    expect(element.attr('aria-label')).toBe('test 2');
+    ariaLabelledby2 = element.attr('aria-labelledby');
+    expect(ariaLabelledby2).toEqual(ariaLabelledby);
+  });
+
+  it('should not update the parent aria-label when the interpolated' +
+      'value changes and the parent has a label that was not set by ' +
+      'the tooltip', function() {
+    buildTooltip(
+      '<md-button aria-label="Button Label">' +
+        '<md-tooltip>{{ testModel.ariaText }}</md-tooltip>' +
+      '</md-button>'
+    );
+
+    $rootScope.$apply(function() {
+      $rootScope.testModel.ariaText = 'test 1';
+    });
+
+    expect(element.attr('aria-label')).toBe('Button Label');
+    expect(element.attr('aria-labelledby')).toBeUndefined();
+
+    $rootScope.$apply(function() {
+      $rootScope.testModel.ariaText = 'test 2';
+    });
+
+    expect(element.attr('aria-label')).toBe('Button Label');
+    expect(element.attr('aria-labelledby')).toBeUndefined();
+  });
 
-        expect(element.attr('aria-label')).toBe('test 2');
-      });
-  
   it('should not interpolate interpolated values', function() {
+    var ariaLabelledby;
+    var ariaLabelledby2;
+
     buildTooltip(
-        '<md-button>' +
-         '<md-tooltip>{{ testModel.ariaTest }}</md-tooltip>' +
-        '</md-button>'
-      );
+      '<md-button>' +
+       '<md-tooltip>{{ testModel.ariaTest }}</md-tooltip>' +
+      '</md-button>'
+    );
 
-      $rootScope.$apply(function() {
-        $rootScope.testModel.ariaTest = 'test {{1+1}}';
-      });
+    $rootScope.$apply(function() {
+      $rootScope.testModel.ariaTest = 'test {{1+1}}';
+    });
 
-      expect(element.attr('aria-label')).toBe('test {{1+1}}');
+    expect(element.attr('aria-label')).toBe('test {{1+1}}');
```

Comment:
@clshortfuse I agree with you. However, the panel API is not technically putting the tracked ID on the panel DOM element. This would make the `aria-labelledby` attribute not function properly. @ErinCoughlan and I are deliberating on this issue and will be creating a new issue for it soon.